docs/contributing/common-scenarios.html

```diff
@@ -0,0 +1,94 @@
+{{>header}}
+
+{{>nav this}}
+
+	<div class="o-techdocs-main o-techdocs-content">
+
+		<h1>Common contribution scenarios</h1>
+
+		<p>A few scenarios come up time and again on our issue tracker. If you want to contribute, and one of the scenarios below descirbes what you want to change, follow the steps to acheive a painless and rapidly merged pull request!</p>
+
+
+		<h3 id='adding-browsers'>A particular polyfill is being served to the wrong browsers / not enough browsers</h3>
+
+		<p>Commonly, people want to take a polyfill we already have and get us to serve it to a browser that we currently don't target with that polyfill. Often this is because it's a less common browser and we just haven't tested the polyfill in that browser, so adding the extra browser as a target does make sense.</p>
+
+		<ol>
+			<li>Update the relevant polyfill's <code>config.json</code> file to add, remove or change the value of the browser version constraint</li>
+			<li>Check that your change is in line with the documentation on writing <a href='/v{{apiversion}}/docs/contributing/authoring-polyfills#configuration'>configuration JSON</a></li>
+			<li>Open a pull request with the following information</li>
+			<li>Check reliable sources of browser support information, eg MDN, caniuse and Kangax.  Tell us what these sources say about the browser/polyfill combo that you want to add or remove, and link to the relevant pages from your PR.</li>
+			<li>Explain how you have tested support (or lack of) in the browser concerned.  Ideally, include a screenshot of the relevant browser running our test suite for that feature.</li>
+			<li>State your agreement to our contribution terms.</li>
+		</ol>
+
+
+		<h3 id='new-browser'>The service is incorrectly identifying a browser</h3>
+
+		<p>Although there are only five significant rendering engines (blink, presto, trident, gecko and webkit), there are lots of distinct browsers and user agents. We don't currently support anywhere near all of them, but are happy to add support on demand.</p>
+
+		<ol>
+			<li>Find out the <code>User-Agent</code> header value sent by the browser</li>
```

Comment:
include a site that will tell you your user agent such as http://www.whatsmyua.com/